resources/podTemplates/terraform-agent.yaml

```diff
@@ -49,6 +49,17 @@ spec:
         name: "workspace-volume"
         readOnly: false
       workingDir: "/home/jenkins/agent"
+    - name: snyk
+      image: artifactory.cloud.cms.gov/docker/snyk/snyk:maven-3-jdk-11
```

Comment:
Is the maven image the one we want for terraform? If it doesn't matter lets use alpine to match what terraform is using.

Reply:
Changing to alpine image